server/spinwick.go

```diff
@@ -142,8 +142,8 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 	return installationID, false, nil
 }
 
-func (s *Server) handleUpdateSpinWick(pr *model.PullRequest) {
-	installationID, sendMattermostLog, err := s.updateSpinWick(pr)
+func (s *Server) handleUpdateSpinWick(pr *model.PullRequest, withLicense bool) {
```

Comment:
Is there a reason we don't want to just give every spinwick a license? I was wondering this earlier, but now I am thinking it may be best to keep things simple and allow for non-HA spinwicks to test other license features.

Reply:
@gabrieljackson for non ha, you might want to test without license and then later add a license to test other things. but i'm 0/5 to add a license to all

@jwilander any comments?

Reply:
I'm fine with giving them all a license

Reply:
0/5 though, we can do that in a later PR

Reply:
Same, only like 1/5. I was just thinking it would keep things simple in the mattermod code while we work on the other bugs.

Reply:
Same, only like 1/5. I was just thinking it would keep things simple in the mattermod code while we work on the other bugs.

Reply:
Same, only like 1/5. I was just thinking it would keep things simple in the mattermod code while we work on the other bugs.

Reply:
i will update